code/datums/datum.dm

```diff
@@ -11,6 +11,28 @@
     var/last_find_references = 0
 #endif
 
+//Return TRUE to prevent clickcode execution.
+/datum/proc/InterceptClickOn(mob/M, atom/target, params)
+	return FALSE
+
+/datum/proc/InterceptDblClickOn(mob/M, atom/target, params)
+	return FALSE

```

Comment:
C O M P O N E N T I Z E

Reply:
This, also most of these are still client persistent so that needs to be supported.